core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1253,6 +1248,23 @@ private String printDynamicFilterAssignments(Map<DynamicFilterId, Symbol> filter
                     .collect(Collectors.joining(", ", "{", "}"));
         }
 
+        private void printInputInfo(NodeRepresentation nodeOutput, PlanNodeStats nodeStats)
+        {
+            if (nodeStats!= null) {
+                double filtered = 100.0d * (nodeStats.getPlanNodeInputPositions() - nodeStats.getPlanNodeOutputPositions()) / nodeStats.getPlanNodeInputPositions();
```

Comment:
`filtered` is applicable only to ScanFilterProject and not TableScan, so I'd suggest not trying to make a common function out of it